packages/contracts/src/dollar/interfaces/IERC1155Ubiquity.sol

```diff
@@ -6,13 +6,23 @@ import "@openzeppelin/contracts/token/ERC1155/IERC1155.sol";
 /// @title ERC1155 Ubiquity preset interface
 /// @author Ubiquity DAO
 interface IERC1155Ubiquity is IERC1155 {
-    function mint(address to, uint256 id, uint256 amount, bytes memory data) external;
+    function mint(address to, uint256 id, uint256 amount, bytes memory data)
+        external;
 
-    function mintBatch(address to, uint256[] memory ids, uint256[] memory amounts, bytes memory data) external;
+    function mintBatch(
+        address to,
+        uint256[] memory ids,
+        uint256[] memory amounts,
+        bytes memory data
+    ) external;
 
     function burn(address account, uint256 id, uint256 value) external;
 
-    function burnBatch(address account, uint256[] memory ids, uint256[] memory values) external;
+    function burnBatch(
+        address account,
+        uint256[] memory ids,
+        uint256[] memory values
+    ) external;
```

Comment:
Can you explain the formatting issue here? I thought we had formatting fixed already.

Reply:
@rndquu can you look into this issue?

Reply:
> @rndquu can you look into this issue?

the PR is merged so I guess you resolved the issue

anyway the PR looks fine

Reply:
> @rndquu can you look into this issue?

the PR is merged so I guess you resolved the issue

anyway the PR looks fine

Reply:
I'm asking about the formatter. We shouldn't have two ways to format the code (one way is incorrect.)

Reply:
Prettier runs only on a local developer machine for staged files.

At some point in the past the linter was broken so there were commits without prettier formatting.

If you run `yarn prettier:all` right now you will see that prettier wants to update 7 files with a proper formatting.

I think they will all eventually be formatted in other issues/PRs so it is not a big concern right now.

Reply:
It's important to have this handled because it takes a lot more effort to review pull requests when I have to parse through hundreds of lines of white spaces changes. Would you mind removing the other way to format so that there's only one way to do it, and ideally, it should be automatic as a pre push hook?

Reply:
> It's important to have this handled because it takes a lot more effort to review pull requests when I have to parse through hundreds of lines of white spaces changes. Would you mind removing the other way to format so that there's only one way to do it, and ideally, it should be automatic as a pre push hook?

ok, I marked myself "spec better prettier"

Reply:
I presume that means you will do the research and create a bounty for the fixed formatting. If its faster to just do it you should probably just implement the fix yourself though. e.g. it could be an exception here to write a very lean spec and then take the bounty if its something you can do in like an hour.

Reply:
> I presume that means you will do the research and create a bounty for the fixed formatting. If its faster to just do it you should probably just implement the fix yourself though. e.g. it could be an exception here to write a very lean spec and then take the bounty if its something you can do in like an hour.

ok